.gitignore

```diff
@@ -13,3 +13,58 @@ src/config.mlh
 .direnv
 .rustup
 result
+isolate*-v8.log
```

Comment:
Can you move these to your dev environment's global gitignore, rather than the project's?